README.md

```diff
@@ -47,7 +47,7 @@ These settings work out-of-the-box with most C64 Ultimate setups. You can overri
 
 ### Easy Installation 📦
 
-Grab the right package for your system from the [Releases page](../../releases):
+Grab the right package for your system from the [Releases page](https://github.com/chrisgleissner/c64u-obs/releases):
```

Comment:
@cursoragent Not sure why this needs to be a bit absolute link. Revert if possible.